app/views/Head.scala.html

```diff
@@ -23,6 +23,3 @@
 <!--[if gt IE 8]><!--><link href='@controllers.routes.Assets.versioned("stylesheets/application.css")' media="all" rel="stylesheet" type="text/css" /><!--<![endif]-->
  <script src='@controllers.routes.Assets.versioned("lib/govuk-frontend/govuk/all.js")'></script>
  <script src='@controllers.routes.Assets.versioned("lib/hmrc-frontend/hmrc/all.js")'></script>
- <script>window.GOVUKFrontend.initAll();</script>
- <script>window.HMRCFrontend.initAll();</script>
-
```

Comment:
There's a similar code in penalties-frontend where we have duplicate initAll() [https://github.com/hmrc/penalties-frontend/blob/7497cc9f828985cf927fcd4217b16c964a1393d7/app/views/Head.scala.html#L26](url)

Do we want them here.

Reply:
I'll raise a PR for the changes on the penalties-frontend service, good spot

Reply:
Here you go: https://github.com/hmrc/penalties-frontend/pull/145/files

Reply:
The character count used in make-a-late-appeal page doesn't seem to be working, I guess it would need the script window.HMRCFrontend.initAll()

Reply:
It didn't work for me when I first opened up this PR. Do you have the latest commit?

Reply:
It works for me on the /making-a-late-appeal:
![image](https://user-images.githubusercontent.com/53828896/151806260-e6910044-fb20-4ac5-9480-7f94614a20a4.png)

Reply:
Yes sure, thanks, I can see them working now. Apparently I wasn't testing on your latest commit, apologies.